### ISSUE TITLE
ReportDownloadTimeout not set.

### ISSUE THREAD

**Issue by mhousley** [googleads/googleads-java-lib#106]
Editing reportDownloadTimeout in an ads.properties file does not set this parameter when using ReportDownloadInterface in client version >=3.1.0. To reproduce:

Set api.adwords.reportDownloadTimeout in ads.properties to a non-default value.

Add the line:
System.out.println(reportDownloader.getReportDownloadTimeout());
to
https://github.com/googleads/googleads-java-lib/blob/master/examples/adwords_axis/src/main/java/adwords/axis/v201702/reporting/DownloadCriteriaReportWithSelector.java

Run the example. The default value 180000 prints to the console.

Workaround: override by using .setReportDownloadTimeout after reportDownloader is instantiated.

**Comment by jradcliff** [comment]
Hi,

Hmmm...I just tried this out locally, and I see the change to `api.adwords.reportDownloadTimeout` reflected in the `println` statement.

Could you confirm the following for the `ads.properties` file where you made this change?

1. The file is in your home directory.
1. The file contains only *one* line for `api.adwords.reportDownloadTimeout`.

Thanks,
Josh, AdWords API Team

**Comment by mhousley** [comment]
It looks like I made some unwarranted assumptions when I was initially testing this. I point the example to a non-default ads.properties by passing an argument into .fromFile() for the Credential and AdWordsSession:
```
    // Generate a refreshable OAuth2 credential.
    Credential oAuth2Credential = new OfflineCredentials.Builder()
        .forApi(Api.ADWORDS)
        .fromFile("ads.properties.prod")
        .build()
        .generateCredential();

    // Construct an AdWordsSession.
    AdWordsSession session = new AdWordsSession.Builder()
        .fromFile("ads.properties.prod")
        .withOAuth2Credential(oAuth2Credential)
        .build();
```
The session is created from the credential and clientCustomerId in ads.properties.prod, but the timeout parameter for the ReportDownloaderInterface is picked up from ads.properties in my home directory.

**Comment by jradcliff** [comment]
Hi,

Thanks for the update. I'll keep this issue open, because I think the assumption you were making about the timeout coming from the non-default file you specified is reasonable. I'll look into what we can do to make that work as you expected.

Cheers,
Josh, AdWords API Team

**Comment by jradcliff** [comment]
Hi,

This was fixed in [version 3.5.0](https://github.com/googleads/googleads-java-lib/releases/tag/3.5.0).

Cheers,
Josh, AdWords API Team
